components/brave_news/common/features.cc

```diff
@@ -32,9 +32,9 @@ const base::FeatureParam<double> kBraveNewsInlineDiscoveryRatio{
     &kBraveNewsFeedUpdate, "inline-discovery-ratio", 0.25};
 
 const base::FeatureParam<double> kBraveNewsSourceSubscribedBoost{
-    &kBraveNewsFeedUpdate, "source-subscribed-boost", 1};
```

Comment:
Do we want to make these the same @LorenzoMinto? Shouldn't subscribing to a source be more of a boost than to a channel? Or does the normalization here make things more clear (because publishers have fewer articles than channels).

Reply:
The reasoning for that is that we don't want to oversample a particular source. Having the same boosting for channel and publisher means that the articles from a publisher will be shown _as often_ as all the articles from a channel, where possible.